src/test/java/amazonreviewers/project/service/CsvFileReaderImplTest.java

```diff
@@ -43,22 +72,40 @@ public void before() {
         listForList.add(firstList);
         listForList.add(secondList);
         listForList.add(thirdList);
+
+        commentDto = new CommentDto();
+        commentDto.setId(ID);
+        commentDto.setProductId(PRODUCT_ID);
+        commentDto.setUserId(USER_ID);
+        commentDto.setProfileName(PROFILE_NAME);
+        commentDto.setHelpfulnessNumerator(HELPFULNESS_NUMERATOR);
+        commentDto.setHelpfulnessDenominator(HELPFULNESS_DENOMINATOR);
+        commentDto.setScore(SCORE);
+        commentDto.setTime(TIME);
+        commentDto.setSummary(SUMMARY);
+        commentDto.setText(TEXT);
     }
 
     @Test(expected = NoSuchElementException.class)
     public void isEmptyFileTest() {
         createReader().readFile(PATH_EMPTY_LIST);
     }
 
-    @Test(expected = NoHeaderException.class)
-    public void fileWithoutHeader() {
-       createReader().readFile(LIST_WITHOUT_HEADER);
-    }
-
     @Test
     public void isNotEmptyFileTest() {
         Assert.assertEquals(listForList,
                 createReader()
                         .readFile(NOT_EMPTY_LIST));
     }
+
+    @Test(expected = NoHeaderException.class)
+    public void fileWithoutHeader() {
+        createDataParser().parseData(LIST_WITHOUT_HEADER);
+    }
```

Comment:
why are you testing both reader and parser in the same test class? they should be tested in separate classes